src/index.html

```diff
@@ -7,8 +7,50 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <div class="spacer"></div>
+      <nav>
+        <ul>
+          <li>
+            <a href="#" class="is-active">
```

Comment:
Please apply my previous suggestions, try to do something like this:
```suggestion
            <a href="#" class="nav__link nav__link--active">
```

Reply:
Active and inactive links should have common styles in the same selector (nav__link). Additional styles for active link should be put inside of nav__link--active